airflow/providers/http/operators/http.py

```diff
@@ -64,6 +64,7 @@ class SimpleHttpOperator(BaseOperator):
         'data',
         'headers',
     ]
+    template_fields_renderers = {'headers': 'json', 'data': 'py'}
```

Comment:
Can you provide the screen of data being rendered with `py` lexer?

Reply:
Hi @turbaszek ,
Please find attached screenshots of template_fields_renderers dict and rendered template for both data & header:
<img width="724" alt="field_description" src="https://user-images.githubusercontent.com/40456773/96181098-c05cfb80-0f33-11eb-9b61-f5a98e05e383.png">
<img width="873" alt="rendered_template" src="https://user-images.githubusercontent.com/40456773/96181106-c2bf5580-0f33-11eb-9aa1-5330ff98de36.png">

Reply:
Thanks @irudson! Looks good to me 👍